src/modules/referrals/components/ProjectOutgoingReferralPostingsTable.tsx

```diff
@@ -105,6 +64,47 @@ const ProjectOutgoingReferralPostingsTable: React.FC<Props> = ({
     ReferralPostingStatus.DeniedStatus,
   ]);
 
+  const rowSecondaryActionConfigs = useCallback(
+    ({ id, project, hohEnrollment }: OutgoingReferral) => {
+      if (!project) return []; // user does not have access to the receiving project
+
+      // If we have a hohEnrollment (meaning the referral was accepted), link to it.
+      // NOTE: its possible that "hohName" and the actual person on "hohEnrollment" don't actually match up,
+      // if the Hoh was changed over time. Thats probably OK, the user can at least get to the right household.
+
+      const enrollmentLinks = hohEnrollment
+        ? [
+            {
+              title: 'View Enrollment',
+              key: 'referral',
```

Comment:
Key should be `enrollment` rather than `referral`, for uniqueness, I think. 

Maybe we could use the [`getViewEnrollmentMenuItem` util](https://github.com/greenriver/hmis-frontend/blob/release-151/src/components/elements/table/tableRowActionUtil.tsx#L36-L49) for consistency. It looks like that util requires an `entryDate` which this fragment doesn't have, which could probably be worked around with some typescript finagling, but, no worries if you want to consider that out of scope.

Reply:
thanks! and good call-out